comments/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django_comments_xtd.models import XtdComment
+
+
+class AllowCommentsModelMixin(models.Model):
```

Comment:
would consider renaming this to `CommentsMixin` or `CommentableMixin`.

what happens if other attributes are required in this class e.g. time-restricted commenting?

Reply:
Shouldn't 'other required attributes' be set in the setting panels?